app/views/layout.njk

```diff
@@ -20,7 +20,7 @@
   {% set serviceName = service.serviceName.en %}
 {% endif %}
 
-{% set headerText = product.name if product and product.type === 'ADHOC' and product.newPaymentLinkJourneyEnabled else serviceName %}
+{% set headerText = product.name if product and product.type === 'ADHOC' and (product.newPaymentLinkJourneyEnabled or NEW_PAYMENT_LINK_JOURNEY_ENABLED_FOR_ALL_PAYMENT_LINKS) else serviceName %}
```

Comment:
Is the env variable available to nunjucks template by default or do we need to set the data in controller?